packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsDescription.tsx

```diff
@@ -1,16 +1,19 @@
 import React, { Fragment } from 'react';
+import { useRegulationFlags } from '@/hooks';
+import { useActiveTradingAccount } from '@deriv/api';
 import { Text } from '@deriv-com/ui';
 import { THooks } from '../../../../types';
 import { getJurisdictionDescription } from './CompareAccountsConfig';
 
 type TCompareAccountsDescription = {
-    isDemo: boolean;
-    isEuRegion: boolean;
     marketType: THooks.AvailableMT5Accounts['market_type'];
     shortCode: THooks.AvailableMT5Accounts['shortcode'];
 };
 
-const CompareAccountsDescription = ({ isDemo, isEuRegion, marketType, shortCode }: TCompareAccountsDescription) => {
+const CompareAccountsDescription = ({ marketType, shortCode }: TCompareAccountsDescription) => {
+    const { data: activeTrading } = useActiveTradingAccount();
+    const { isEU: isEuRegion } = useRegulationFlags();
```

Comment:
Reason of replacing this cause useRegulationFlags already uses this hook and isEU coming from useRegulationFlags also takes account of the EU switcher condition for DIEL countries. So two birds one stone 😺